src/cascadia/TerminalApp/CascadiaSettings.cpp

```diff
@@ -643,3 +643,128 @@ Profile CascadiaSettings::_CreateDefaultProfile(const std::wstring_view name)
 
     return newProfile;
 }
+
+// Method Description:
+// - Gets our list of warnings we found during loading. These are things that we
+//   knew were bad when we called `_ValidateSettings` last.
+// Return Value:
+// - a reference to our list of warnings.
+std::vector<TerminalApp::SettingsLoadWarnings>& CascadiaSettings::GetWarnings()
+{
+    return _warnings;
+}
+
+// Method Description:
+// - Attempts to validate this settings structure. If there are critical errors
+//   found, they'll be thrown as a SettingsLoadError. Non-critical errors, such
+//   as not finding the default profile, will only result in an error. We'll add
+//   all these warnings to our list of warnings, and the application can chose
+//   to display these to the user.
+// Arguments:
+// - <none>
+// Return Value:
+// - <none>
+void CascadiaSettings::_ValidateSettings()
+{
+    _warnings.clear();
+
+    _ValidateProfilesExist();

```

Comment:
Is the order of these very important? Will there be trouble if someone inserts one in the wrong spot in the future or reorders them?